docs/data/data-grid/aggregation/aggregation.md

```diff
@@ -4,40 +4,40 @@ title: Data Grid - Aggregation
 
 # Data Grid - Aggregation [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
 
-<p class="description">Add aggregation functions to the Data Grid so users can combine row values.</p>
+<p class="description">Add aggregation functions to the Data Grid to let users combine row values.</p>
 
-You can aggregate rows through the Data Grid interface by opening the column menu and selecting from the items under **Aggregation**.
+The Data Grid Premium provides tools to give end users the ability to aggregate and compare row values.
+It includes [built-in functions](#built-in-functions) to cover common use cases such as sum, average, minimum, and maximum, as well as the means to [create custom functions](#creating-custom-functions) for all other needs.
```

Comment:
not sure if there should be a `,` here

```suggestion
It includes [built-in functions](#built-in-functions) to cover common use cases such as sum, average, minimum and maximum, as well as the means to [create custom functions](#creating-custom-functions) for all other needs.
```

Reply:
That's called an Oxford (or serial) comma, and I'm a big fan of it. 😁 It seems like a very minor detail but English can be a pretty ambiguous language so I always prefer to err on the side of clarity. In one case, the lack of an Oxford comma in an employment contract led to a successful [multi-million-dollar lawsuit](https://archive.ph/Ggz0x)!